programmer-humour/script.js

```diff
@@ -0,0 +1,19 @@
+function accessToAPI() {
+  return fetch("https://xkcd.now.sh/?comic=latest").then((res) => {
+    if (!res.ok) {
+      return "Error!";
```

Comment:
You could return a JSON structure here.  You code in `render` will be expecting `data` to be returned, so you should check whether error data is returned.